src/components/helpers/figure_container_helper.ts

```diff
@@ -0,0 +1,40 @@
+import { DOMCoordinates, Figure, Getters } from "../../types";
+
+function internalToScreenCoordinates(getters: Getters, { x, y }: DOMCoordinates): DOMCoordinates {
```

Comment:
"screen" here refers to the full sheet view (main viewport + freezed) ?

Reply:
It's explained in the docString of `internalFigureToScreen`, which is the exposed method.